pkg/cmd/grafana-cli/commands/install_command.go

```diff
@@ -125,7 +125,7 @@ func doInstallPlugin(ctx context.Context, pluginID, version string, o pluginInst
 		if p, ok := services.PluginVersionInstalled(pluginID, version, o.pluginDir); ok {
 			services.Logger.Successf("Plugin %s v%s already installed.", pluginID, version)
 			for _, depP := range p.JSONData.Dependencies.Plugins {
-				if err := doInstallPlugin(ctx, depP.ID, depP.Version, o, installing); err != nil {
+				if err := doInstallPlugin(ctx, depP.ID, "", o, installing); err != nil {
```

Comment:
feel free to disregard this but what if we keep the `Version` property and throw an error if a plugin is attempting to install a dependency with a version? Again, it depends on the popularity of this feature, if it's not used in production we can keep this behavior and ignore if the version it is defined.